presets/4.5/filters/aos_rc/aos_65mm_filters.txt

```diff
@@ -0,0 +1,37 @@
+#$ TITLE: AOS 65mm Filters
+#$ FIRMWARE_VERSION: 4.5
+#$ CATEGORY: FILTERS
+#$ STATUS: COMMUNITY
```

Comment:
agree with @limonspb  should be experimental

Reply:
changed 👍